open_instruct/finetune.py

```diff
@@ -1001,49 +1001,65 @@ def main(args: FlatArguments):
         )
 
     # remove all checkpoints to save space
-    if accelerator.is_main_process:
+    if accelerator.is_local_main_process:
```

Comment:
This should remove the intermedaite checkpoints on all the nodes.

Reply:
Looks good!